projects/ngx-editor/src/lib/editor.module.ts

```diff
@@ -3,6 +3,7 @@ import { CommonModule } from '@angular/common';
 
 import { NgxEditorConfig } from './types';
 import { defaults as defaultLocals } from './Locals';
+import { defaults as defaultIcons } from './Icons';
```

Comment:
```suggestion
import { defaults as defaultIcons } from './Icons';
```

It would be better name this as `icons` or `defaultIcons` to avoid some confusion